charts/prometheus-blackbox-exporter/templates/daemonset.yaml

```diff
@@ -48,7 +48,16 @@ spec:
         - name: {{ . }}
       {{- end }}
     {{- end }}
-
+    {{- if .Values.hostAliases }}
```

Comment:
Could you move the indentation of those lines to respect the implicit convention ?

Reply:
I have gone simply by the other definitions above, but I have changed now the indention.

Reply:
Thanks! Sorry about that, so we have something non-logical, I'll take care of it someday.